plasmapy/plasma/grids.py

```diff
@@ -1184,7 +1184,7 @@ def volume_averaged_interpolator(
                     valid = (
                         (x >= 0) & (x < n0) & (y >= 0) & (y < n1) & (z >= 0) & (z < n2)
                     )
-                    out = np.where(valid == False)
+                    out = np.where(not valid)
```

Comment:
This actually crashes the tests ( https://github.com/PlasmaPy/PlasmaPy/pull/1127/checks?check_run_id=2475959379#step:6:184 ), because `valid` is a numpy array and and the usual pythonic negation doesn't work element-wise by default. This should fix it:
```suggestion
                    out = np.where(~valid)
```